examples/convert_group_studies.py

```diff
@@ -16,30 +16,40 @@
 
 # Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
 #          Teon Brooks <teon.brooks@gmail.com>
+#          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
 
 # License: BSD (3-clause)
 
 ###############################################################################
 # Let us import ``mne_bids``
 
 import os.path as op
+import subprocess
 
 import mne
 from mne_bids import write_raw_bids, make_bids_basename
-from mne_bids.datasets import fetch_faces_data
 from mne_bids.utils import print_dir_tree
 
 ###############################################################################
 # And fetch the data.
-# .. warning:: This will download 7.9 GB of data for one subject!
+# .. warning :: This will download 11.8 GB of data!
 
-subject_ids = [1]
-runs = range(1, 7)
-
-home = op.expanduser('~')
-data_path = op.join(home, 'mne_data', 'mne_bids_examples')
 repo = 'ds000117'
-fetch_faces_data(data_path, repo, subject_ids)
+data_address = 's3://openneuro.org/{}/'.format(repo)
+
+data_path = op.join(op.expanduser('~'), 'mne_data', 'mne_bids_examples', repo)
+
+# Prepare the aws command
+cmd = ['aws', 's3', 'sync', '--no-sign-request', data_address, data_path,
+       '--exclude', '*']
+
+subject_ids = [1, 2]
+for subject_id in subject_ids:
+    cmd += ['--include', 'sub-{:02}*'.format(subject_id)]
+
+# Download
+subprocess.run(cmd)
```

Comment:
Instead of downloading BIDS data here and formatting to BIDS again ... I could use an MNE fetcher --> again, this would allow me to not use `awscli` and still drop all `datasets.py` module dependencies.

E.g. `mne.datasets.spm_face.data_path()`

Reply:
also for this one: Any opinions @jasmainak @agramfort ?